app/assets/stylesheets/templates/_ctc-home.scss

```diff
@@ -18,10 +18,34 @@
     padding-bottom: 0;
   }
 
-  .slab--why {
+  .slab--affiliations {
     background-color: $color-grey-light;
   }
 
+  .affiliations {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+
+    .grid__item {
+      display: flex;
+      flex-direction: column;
+      align-items: center;
+    }
+    .grid__item.text {
+      width: 200px;
+      text-align: center;
+    }
+    .grid__item.image {
```

Comment:
if you make this a flex item, you can center the image tag within the div so that hopefully all of the images will look more balanced on the page -- it might pull down the code for america logo so that it aligns better with the center of the white house image.